frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -1094,7 +1094,7 @@ const ManageHostsPage = ({
     try {
       await (isAllMatchingHostsSelected
         ? hostsAPI.destroyByFilter({
-            teamId,
+            teamId: teamIdForApi,
```

Comment:
above `teamId` definition is now unnecessary right?

Reply:
100% thanks for catching this